cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/auth/AccessTokenValidator.java

```diff
@@ -0,0 +1,37 @@
+package mil.army.usace.hec.cwms.http.client.auth;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import java.time.Instant;
+import java.util.Date;
+
+public final class AccessTokenValidator {
+
+    private static final String REFRESH_EXPIRED_BUFFER_PROPERTY_KEY = "cwms.http.client.token.refresh.buffer.millis";
+    private static final long DEFAULT_REFRESH_EXPIRED_BUFFER_SECONDS = 1;
```

Comment:
I would bump this up to a minute for safety. The refresh token expiration for cumulus is on the order of hours right?